.pre-commit-config.yaml

```diff
@@ -81,6 +81,7 @@ repos:
           - "@typescript-eslint/eslint-plugin"
           - "@typescript-eslint/parser"
           - eslint-plugin-diff@^2.0.3
+        exclude: ^.*\.(?<!\.d\.)ts$
```

Comment:
i think if you just exclude the ts files in res/translations it would be much clearer